pkg/otlp/map_provider.go

```diff
@@ -42,27 +46,34 @@ func buildMetricsMap(cfg PipelineConfig) (*confmap.Conf, error) {
 	if err != nil {
 		return nil, err
 	}
-
+	smap := map[string]interface{}{
+		buildKey("exporters", "serializer", "metrics"): cfg.Metrics,
+	}
+	if cfg.OpenCensusEnabled {
+		smap[buildKey("service", "pipelines", "metrics", "receivers")] = []interface{}{"otlp", "opencensus"}
+	}
 	{
```

Comment:
I haven't seen this syntax of having `{ }` with no keyword starting it, I'm curious what this means?

Reply:
I'm not sure what you mean? Can you please write a more complete example? Do you mean the `{}` inside `[]interface{}`? It just means a slice of "anything". It could've been `[]string`, but it's clear that from how this map works (by seeing test results) that `[]interface{}` is expected. It's not the most idiomatic approach Go-wise, but it is what it is.